panel/models/chatarea_input.ts

```diff
@@ -31,9 +32,15 @@ export class ChatAreaInputView extends PnTextAreaInputView {
     super.render()
 
     this.el.addEventListener("keydown", (event) => {
-      if (event.key === "Enter" && !event.shiftKey) {
+      console.log( "DBG: keydown event", event.key )
+      if (event.key === "Enter" && (  (!event.shiftKey &&  this.model.enter_sends)
+                                   || ( event.shiftKey && !this.model.enter_sends))
+      ) {
+        console.log( ".  enter_sends:     ", event.shiftKey && !this.model.enter_sends )
```

Comment:
Maybe if you swap it?
```suggestion
  if (event.key === "Enter" && (
    (event.shiftKey && !this.model.enter_sends) ||
    (!event.shiftKey && this.model.enter_sends)
  )) {
    if (!this.model.disabled_enter) {
      console.log(".  .  FIRE: ", this.model.value_input)
      this.model.trigger_event(new ChatMessageEvent(this.model.value_input))
      this.model.value_input = ""
    }
    event.preventDefault()
  }
})
```